.gitignore

```diff
@@ -6,5 +6,9 @@ deps/ext
 test/MockFTPServerJulia.class
```

Comment:
Both `deps/ext` and `test/MockFTPServerJulia.class` can be removed from this file.